maps/tether/tether-02-surface2.dmm

```diff
@@ -20835,16 +20826,8 @@
 	temperature = 80
 	},
 /area/tcommsat/chamber)
-"aNc" = (
-/obj/machinery/telecomms/relay/preset/tether/base_high,
-/turf/simulated/floor/tiled/dark{
-	nitrogen = 100;
-	oxygen = 0;
-	temperature = 80
-	},
-/area/tcommsat/chamber)
 "aNd" = (
-/obj/machinery/telecomms/broadcaster/preset_right,
+/obj/machinery/telecomms/broadcaster/preset_right/tether,
```

Comment:
Will the changes in relays also affect the Underdark/Wilderness comms? Those two are not meant to be connected IIRC

Reply:
They have a mapped in relay for the UD so I'm not sure about that recollection. Plains will be newly connected yes, but it's within walking distance of tether. I dunno why a radio couldn't go that far.